File: number-of-senior-citizens/number-of-senior-citizens.py
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        n = len(details)
        count = 0
        
        for i in details:
            x = int(i[11:13])
            
            if(x > 60):
                count += 1
        
        return count
        

File: 2727-number-of-senior-citizens/number-of-senior-citizens.js
/**
 * @param {string[]} details
 * @return {number}
 */
var countSeniors = function (details) {
    let count = 0;

    for (let i = 0; i < details.length; i++) {
        let x = parseInt(details[i].substring(11, 13), 10);

        if (x > 60) {
            count += 1;
        }
    }

    return count;
};